Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,413 @@
+    , sosAppSignatureID = sid // Application Signature ID
+    , sosIPSCategoryName = ipscat // IPS Category Name
+    , sosAntiSpywareCategory = spycat // Anti-Spyware Category
+    , sosURLPathName = arg // URL. Represents the URL path name.
+    , sosFileIdentifier = fileid // File hash or URL
+    , sosDPIInspectedFlow = dpi // Indicates a flow was inspected by DPI. Applies only to Connection Closed messages.
+    , DstNatPortNumber = dnpt
+    , SrcNatPortNumber = snpt
+    , sosBladeID = bid // Blade ID
+    , sosUUID = uuid
+    , sosHttpRequestMethod = HttpRequestMethod
+    , sosFileName = FileName
+    , DvcOriginalAction = fw_action
+  | extend
+    ThreatName = coalesce(sosASWSignatureName, sosGAVSignatureName, sosIPSSignatureName, "")
+    , ThreatId = coalesce(sosAppSignatureID, "")
+    , ThreatCategory = coalesce(sosIPSCategoryName, sosAntiSpywareCategory, "")
+    , DstNatPortNumber = toint(DstNatPortNumber)
+    , SrcNatPortNumber = toint(SrcNatPortNumber)
+  | extend AdditionalFields = bag_pack(
+    "ReceivedPackets", sosReceivedPackets
+    , "SentPackets", sosSentPackets
+    , "ConnectionDuration", sosConnectionDuration
+    , "AppRulePolicyId", sosAppRulePolicyId
+    , "AppRulePolicyName", sosAppRulePolicyName
+    , "AppRuleService", sosAppRuleService
+    , "AppRuleType", sosAppRuleType
+    , "AppRuleObject", sosAppRuleObject
+    , "AppRuleObjectContent", sosAppRuleObjectContent
+    , "AppRuleAction", sosAppRuleAction
+    , "AppIDNumber", sosAppIDNumber
+    , "AppID", sosAppID
+    , "AppCategoryID", sosAppCategoryID
+    , "AppSignatureID", sosAppSignatureID
+    , "IPSCategoryName", sosIPSCategoryName
+    , "AntiSpywareCategory", sosAntiSpywareCategory
+    , "URLPathName", sosURLPathName
+    , "FileIdentifier", sosFileIdentifier
+    , "DPIInspectedFlow", sosDPIInspectedFlow
+    , "BladeID", sosBladeID
+    , "UUID", sosUUID
+    , "HttpRequestMethod", sosHttpRequestMethod
+    , "FileName", sosFileName
+    , "FileSize", FileSize
+    , "HttpReferrer", sosHttpReferrer
+    , "HttpUserAgent", HttpUserAgent
+    , "CaptureATPVerdict", sosCaptureATPVerdict
+    , "ApplicationID", sosApplicationID
+    , "CFSCategoryID", sosCFSCategoryID
+    , "CFSCategoryName", sosCFSCategoryName
+    , "CFSPolicyName", sosCFSPolicyName
+    , "AppControlFileName", sosAppControlFileName
+    , "AppCategoryName", sosAppCategoryName
+    , "AppIDName", sosAppIDName
+    , "AppSignatureName", sosAppSignatureName
+    , "IPSFullString", sosIPSFullString
+    , "IPSSignatureName", sosIPSSignatureName
+    , "LegacyMessageCategory", sosLegacyMessageCategory
+    , "LogMsgCategory", sosLogMsgCategory
+    , "LogMsgNote", sosLogMsgNote
+    , "LogMsgSeverity", sosLogMsgSeverity
+    , "SourceVPNPolicyName", sosSourceVPNPolicyName
+    , "DestinationVPNPolicyName", sosDestinationVPNPolicyName
+    , "SerialNumber", sosSerialNumber
+    , "DeviceModel", sosDeviceModel
+    , "LogMsgID", sosLogMsgID

```

Comment:
https://learn.microsoft.com/en-us/azure/sentinel/normalization-common-fields#eventoriginaluid

Reply:
I don't think it fits. The 'LogMsgID' is NOT unique. Each log event has an ID number assigned to it. So if the EventOriginalUid must be unique, our log message IDs should not go there. I didn't see a field that I felt was a proper fit. Maybe you know of one.

![image](https://github.com/Azure/Azure-Sentinel/assets/42151366/a78217e5-c96a-4c78-9aa5-a9be9dabce3f)

![image](https://github.com/Azure/Azure-Sentinel/assets/42151366/ff341ff4-c17b-49e1-b100-dc641c2fe150)

Reply:
Would "EventOriginalType" work? Doc says its a "type or ID", and an example usage is to store the original Windows Event ID, which is a good way to describe our Log IDs. I think this field, while not the best name, is a good match. What do you think?

![image](https://github.com/Azure/Azure-Sentinel/assets/42151366/6d1a2d60-3f46-45ab-8e40-d4cceaed3b9d)

Reply:
@jaimeesc looks good to me. You can map with EventOriginalType

Reply:
@jaimeesc I've remove the , "LogMsgID", sosLogMsgID mapping from AdditionalFields. Also, mapped EventOriginalType = DeviceEventClassID